app/main.py

```diff
@@ -1,11 +1,12 @@
-from typing import List, Annotated
+from typing import List
 
 from starlette.middleware.cors import CORSMiddleware
 
 from app.lib.sql import *
 
-from fastapi import FastAPI, Response, File, UploadFile
-from app.models import MemberIn, MemberOut, Category, CategoryOut, MemberWithCategory, MemberHasCategoryIn, GetMemberHasNetwork, MemberById, Network, MemberHasNetwork, MemberHasCategory, MemberHasNetworkIn
```

Comment:
Why not using imports with parentheses ? 
```py
from app.models import (
    MemberIn,
    MemberOut,
    Category,
    CategoryOut,
    MemberWithCategory,
    MemberHasCategoryIn,
    GetMemberHasNetwork,
    MemberById,
    Network,
    MemberHasNetwork,
    MemberHasCategory,
    MemberHasNetworkIn
)
```

or even better, 
```py
from app import models
```
and in the code, for example 

```py
@app.get("/members", response_model=List[models.MemberWithCategory])
async def api_get_members():
    return await get_members()

```